blocks/chat-cta/chat-cta.js

```diff
@@ -1,4 +1,38 @@
+export async function getCtaBody(url) {
+  const parser = new DOMParser();
+  const cta = await fetch(url)
+    .then((resp) => resp.text())
+    .then((text) => parser.parseFromString(text, 'text/html'))
+    .then((doc) => doc.querySelector('.chat-cta'))
+    .catch((err) => console.log(err, 'Failed to load cta'));
+
+  if (!cta) {
+    return null;
+  }
+
+  cta.classList.add('cta-loaded', 'section');
+  return cta;
+}
+
+export async function libsDecorateCta(cta, libsPath) {
+  const { decorateAutoBlock, getConfig } = await import(`${libsPath}/utils/utils.js`);
+  const { default: loadIcons } = await import(`${libsPath}/features/icons.js`);
```

Comment:
You could load these at the same time, might be a little cumbersome. 
```
  const [{ decorateAutoBlock, getConfig }, { default: loadIcons }] = await Promise.all([import(`${libsPath}/utils/utils.js`), import(`${libsPath}/features/icons.js`]);
```

Reply:
Hrrm. Not sure. Maybe something we could test after the fact for performance improvements?

Reply:
Sounds good, I wasn't sure the performance impact since `loadChatCTA` is ran after `loadDelayed` anyways, utils and icons should already be loaded.